scripts/install-libsodium.sh

```diff
@@ -0,0 +1,7 @@
+curl https://download.libsodium.org/libsodium/releases/libsodium-1.0.12.tar.gz \
+    | tar xz --directory lib/
+pushd lib/libsodium-1.0.12/
+./configure
```

Comment:
so this will download the libsodium tar into the (non-existent) folder `/scripts/lib`... maybe `/lib` was intended?